lang/po/Firmware_ro.po

```diff
@@ -2578,6 +2578,11 @@ msgstr "Sensibilitate"
 msgid "Mesh bed leveling failed. Please run Z calibration."
 msgstr "Nivelarea patului a eșuat. Rulează Calibrare Z."
 
+#. MSG_SET_READY c=18
+#: ../../Firmware/messages.cpp:104 ../../Firmware/ultralcd.cpp:5204
+msgid "Set Ready"
+msgstr "Set pregătit"
```

Comment:
This is wrong. The translation used the meaning of "set" as in "a set of items", which is wrong.
"Printer pregătit" might be better in this context, but it's still not that great.